.github/jobs/configure-checks/all.bats

```diff
@@ -67,9 +67,9 @@ repo-remove () {
     assert_line "checking for gcc... no"
     assert_line "checking for cc... no"
     assert_line "checking for cl.exe... no"
-    assert_line "configure: error: in \`${test_path}':"
+    assert_regex "configure: error: in [\`']${test_path}':"
```

Comment:
if you want, you can replace the `[\`']` by `.` - this would allow also other characters, but I don't think we care and it is more readable

Reply:
As github screws up my formatting above, this is what I meant: 

```suggestion
    assert_regex "configure: error: in .${test_path}':"
```